newyork.html

```diff
@@ -98,66 +109,137 @@ <h6>March 17, 2014</h6>
   
         <div>
 
-            <h4>Thanks for reading The Times.</h4>
-            <h5>Subscribe to The Times</h5>
-            <p class="paragraph">
+            <h4 class="main-article">Thanks for reading The Times.</h4>
+            <h5 class="main-article">Subscribe to The Times</h5>
+            <p class="main-article">
                Reaching back across 13.8 billion years to the first sliver of cosmic time with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics detected ripples in the fabric of space-time — so-called gravitational waves — the signature of a universe being wrenched violently apart when it was roughly a trillionth of a trillionth of a trillionth of a second old. They are the long-sought smoking-gun evidence of inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct.</p>
 
-               <p>Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered whether it could ever be proved.</p>
+               <p class="main-article">Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered whether it could ever be proved.</p>
 
-               <p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.
+               <p class="main-article">If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.
 
                Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
            
 
```

Comment:
## Add a missing feature
- [ ] Just as pointed out by @Spayco, there will be a need to implement the `Editor's Pick` section on our page as it present on the original page and the project's main objective is to be a clone of the original page. Is this achievable? If so, you might want to implement it using the `aside` tag. I have shared below the comparisons.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|![editor's-pick](https://user-images.githubusercontent.com/22345961/108571532-6a54f600-7321-11eb-9bd9-c0f1390aa8ef.png)|![editor's-pick-clone](https://user-images.githubusercontent.com/22345961/108571543-6fb24080-7321-11eb-8ea7-a887298ca328.png)|